R/checkX13binary.R

```diff
@@ -1,15 +1,15 @@
 checkX13binary <- function(){
   if (.Platform$OS.type == "windows"){    
-    x13.bin <- file.path(path.package("x13binary"), "bin", "x13ashtml.exe")
+    x13.bin <- file.path(system.file("bin", package="x13binary"), "x13ashtml.exe")
```

Comment:
even this is too convoluted.  just do 

`system.file("bin", "x13ashtml.exe", package="x13binary")`

and test for empty string on failure.